docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -181,3 +183,27 @@ Deleting a file storage at an instance level deletes the Nextcloud integration c
 ## Using the integration
 
 Once the file storage is added and enabled for projects, your users are able to take full advantage of the integration between Nextcloud and OpenProject. For more information on how to link Nextcloud files to work packages in OpenProject and access linked work packages in Nextcloud, please refer to [Using the Nextcloud integration](../../../user-guide/nextcloud-integration/).
+
+
+
+## Troubleshooting
+
+On Nextcloud inside the OpenProject integration App when adding the OpenProject host it shows the error **"Please enter a valid OpenProject hostname"**.
```

Comment:
![image](https://user-images.githubusercontent.com/94119622/196941444-565ae490-0625-4277-bbe6-9b48e7ad40ef.png)

i also see that this was corrected now

will do the job

Reply:
Hi @wielinde if you review the WHOLE page, you will see that most of the time the original author used Openproject integration and not Openproject Integration, please decide which version we shall use, just review the TITLE and then you will see :)

Reply:
The official name was changed from "Nextcloud integration" to "Nextcloud Integration" as that looks much more like a product name. I know that because I got the feedback and then I changed the official name here https://apps.nextcloud.com/apps/integration_openproject. So could you please adopt the title everywhere in the docs accordingly, please?